README.md

```diff
@@ -1,73 +1,243 @@
-# oas
-
-Working with OpenAPI definitions is hard. This makes it easier.
-
-[![Build](https://github.com/readmeio/oas/workflows/CI/badge.svg)](https://github.com/readmeio/oas/) [![](https://img.shields.io/npm/v/oas)](https://npm.im/oas)
-
-[![](https://d3vv6lp55qjaqc.cloudfront.net/items/1M3C3j0I0s0j3T362344/Untitled-2.png)](https://readme.com)
+<p align="center">
+  <a href="https://npm.im/oas">
+    <img src="https://raw.githubusercontent.com/readmeio/oas/main/.github/hero.png" alt="oas" />
+  </a>
+</p>
+
+<p align="center">
+  Comprehensive tooling for working with OpenAPI definitions
```

Comment:
I like this way more than "Working with OpenAPI definitions is hard. This makes it easier.".

Reply:
Idea I just had: thoughts on adding an emoji here?

```suggestion
  Comprehensive tooling for working with OpenAPI definitions 🛠️
```